src/app/core/site-footer/site-footer/site-footer.component.scss

```diff
@@ -37,6 +37,7 @@ footer {
     grid-column-gap: 20px;
     grid-row-gap: 20px;
     margin-top: 40px;
+    cursor: pointer;
```

Comment:
czemu tego potrzebujemy?

Reply:
Według mnie to nie jest potrzebne, czemu sugerujemy, że da się kliknąć w miejsca, w które nie da się kliknąć?